.github/workflows/ci.yml

```diff
@@ -96,6 +97,36 @@ jobs:
       if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
       env:
         GITHUB_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        
+  alpine-release:
```

Comment:
I think we need to add this as a `needs` value for the `npm-publish` job below, which will ensure that we only publish when we have `alpine` prebuilds available.

Reply:
Done